platform/linux/mbedtls/network_mbedtls_wrapper.c

```diff
@@ -277,6 +279,10 @@ IoT_Error_t iot_tls_connect(Network *pNetwork, TLSConnectParams *params) {
 
 	mbedtls_ssl_conf_read_timeout(&(tlsDataParams->conf), IOT_SSL_READ_TIMEOUT);
 
+#ifdef AWS_IOT_MQTT_SOCKET_NON_BLOCKING
```

Comment:
(Minor) This could be something like `IOT_NETWORK_SOCKET_NON_BLOCKING` to avoid use of `MQTT` in the file.

Reply:
Will do.